src/tests/Microsoft.PowerFx.Interpreter.Tests/TableTests.cs

```diff
@@ -24,5 +26,84 @@ public async Task TestTableWithEmptyRecords()
             Assert.NotNull(t);
             Assert.Equal(3, ((object[])t).Length);
         }
+
+        [Theory]
+        [InlineData(false)]
+        [InlineData(true)]
+        public void Powerfx_Partial_Collect(bool serialize)
+        {
+            FormulaValue result = Run_Collect_Worflow(serialize);
+            if (!serialize)
+            {
+                Assert.Equal("Blank()", result.ToString());
+            }
+            else
+            {
+                Assert.Equal("Runtime type mismatch", ((ErrorValue)result).Errors[0].Message);
```

Comment:
>Assert.Equal("Runtime type mismatch", ((ErrorValue)result).Errors[0].Message); [](http://example.com/codeflow?start=16&length=78)

Please remove this aseert. After the fixes both cases should result in `Blank()`